### PR TITLE
Add a readme with logo and example

### PR DESCRIPTION
**Pull request by floriangardin**
I added two examples and a short description about the tradeoff between interpretability and performances

### CODE REVIEW

**Comment by datajms** [comment]
LGTM +1

File: README.md [hunk 1]
```diff
@@ -17,18 +17,96 @@
...
+                     n_estimators=30,
+                     precision_min=0.2,
+                     recall_min=0.01,
+                     feature_names=dataset.feature_names)
+    
+    X, y = dataset.data, dataset.target > 25
+    X_train, y_train = X[:len(y)//2], y[:len(y)//2]
+    X_test, y_test = X[len(y)//2:], y[len(y)//2:]
+    clf.fit(X_train, y_train)
+    y_score = clf.score_top_rules(X_test) # Get a risk score for each test example
```
Comment (agramfort) [inline README.md]: why not using the standard method `score_samples` from sklearn API? this `score_top_rules` name makes the API very specific to the estimator which defeats the API consistency of sklearn.

Comment (ngoix) [reply README.md]: we have 3 different scoring methods: `decision_function`, `rules_vote`, `score_top_rules`.
Maybe we should add a class parameter to chose one of these 3 functions at initialization ?

**Comment by agramfort** [comment]
hum ok. Is that document somewhere when to use which?

**Pull request merged**